### DISCUSSION TITLE
How to change style and content of default Tooltip in Forms / for FormControl?

### DISCUSSION THREAD

**Discussion by Poolshark** [chakra-ui/chakra-ui#4579] [Q&A] [answered]
Hi,

I was checking the docs and the discussion here but I could not find any information of how to change the default style and content of the tooltip which shows up if `FormContrrol` is set to `required`. Any tips for me here?

Thanks!

<img width="704" alt="Screenshot 2021-08-16 at 13 47 09" src="https://user-images.githubusercontent.com/9102922/129566204-ce457bd8-29d7-4c5d-8fe0-eaa14b5f5aba.png">

**Comment by aaronkik** [comment, marked as answer]
I believe the tooltip you're showing is baked into the browser and cannot be styled. You can do a quick google search for "html required popup".

You could omit the isRequired prop from the FormControl and handle your validation through JS frameworks like [Formik](https://formik.org/) or [react-hook-form](https://react-hook-form.com/). Then perhaps show a custom popup when an error has occured on a given input field.

Chakra example using Formik: https://chakra-ui.com/docs/form/form-control#usage-with-form-libraries

Chakra example using react-hook-form: https://chakra-ui.com/guides/integrations/with-hook-form
